### ISSUE TITLE
About the speed of the image rendering

### ISSUE THREAD

**Issue by wangkke** [ethliup/UnrealCV#1]
Thanks for your sharing very much!  I can now use this dataset !
And i would like to know how long will you spend to render the RS image?  Because when I use the dataset and the engine to render RS image of  900 * 900, it will take about 30 minutes. And i think it's too long. 
I wonder if this is related to the size of GPU?  My GPU memory is 2G.
Thanks for your reply!!

**Comment by ethliup** [comment]
Yes, it is a bit slow but not that slow. It takes less than/roughly 1 min for a 640x480 images. Maybe it is caused by your GPU. I used a GTX1080Ti (12 GB) for the rendering.

**Comment by wangkke** [comment]
Thank you for your reply and wish you a happy life！

> Yes, it is a bit slow but not that slow. It takes less than/roughly 1 min for a 640x480 images. Maybe it is caused by your GPU. I used a GTX1080Ti (12 GB) for the rendering.

Thank you for your reply and wish you a happy life
